src/main/java/seedu/jimi/logic/commands/ShowCommand.java

```diff
@@ -31,14 +31,15 @@
             "thursday", 
             "friday", 
             "saturday", 
-            "sunday" 
+            "sunday",
+            "all"
     );
     
     public static final String MESSAGE_USAGE = 
-            COMMAND_WORD + ": Shows certain sections of the task panel in the agenda panel. \n"
+            COMMAND_WORD + ": Shows certain sections of the task panel or all tasks and events in the agenda panel. \n"
             + "Parameters: NAME_OF_SECTION_TO_DISPLAY\n" 
             + "Example: " + COMMAND_WORD + " floating tasks\n"
-            + "> Valid case-insensitive keywords: Floating, Complete, Incomplete, Today, Tomorrow, {day of week displayed}";
+            + "> Valid case-insensitive keywords: All, Floating, Complete, Incomplete, Today, Tomorrow, {day of week displayed}";
 
     public static final String MESSAGE_SUCCESS = "Displayed tasks and events.";
```

Comment:
Is it possible to make more message_successes?

maybe a hashmap with the same ListId from flm i.e. HashMap<ListId, String> MESSAGE_SUCCESS_MAP

then when command is successfull, return CommandResult(MESSAGE_SUCCESS_MAP.get(userInput)) or something